firebase-appdistribution/src/test/java/com/google/firebase/appdistribution/impl/TesterApiHttpClientTest.java

```diff
@@ -254,13 +256,15 @@ public void makeUploadRequest_writesRequestBodyAndSetsCorrectHeaders() throws Ex
     when(mockHttpsURLConnection.getInputStream()).thenReturn(responseInputStream);
     ByteArrayOutputStream requestBodyOutputStream = new ByteArrayOutputStream();
     when(mockHttpsURLConnection.getOutputStream()).thenReturn(requestBodyOutputStream);
-    File testRequestBodyFile =
-        ApplicationProvider.getApplicationContext().getFileStreamPath("requestBody.txt");
-    try (FileWriter writer = new FileWriter(testRequestBodyFile)) {
-      writer.write("Test post body");
-    }
 
-    testerApiHttpClient.makeUploadRequest(TAG, TEST_PATH, TEST_AUTH_TOKEN, testRequestBodyFile);
+    ContentResolver contentResolver =
+        ApplicationProvider.getApplicationContext().getContentResolver();
+    ShadowContentResolver shadowContentResolver = shadowOf(contentResolver);
+    ByteArrayInputStream inputStream = new ByteArrayInputStream("Test post body".getBytes(UTF_8));
```

Comment:
nit: try-with-resources (it's not really necessary to close a ByteArrayInputStream - especially in tests - but I try to make sure to always close stream - everywhere to avoid accidental resource leaks. I think it's a worthwhile habit.